homeassistant/components/shelly/sensor.py

```diff
@@ -348,7 +348,7 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
     "temperature": RpcSensorDescription(
         key="switch",
         sub_key="temperature",
-        name="Temperature",
+        name="Device Temperature",
```

Comment:
This was changed to be aligned with Gen1 devices and to avoid confusing with the external sensors
https://github.com/home-assistant/core/blob/35a47d83a48fc410f51fd5297f93e95291cd3dff/homeassistant/components/shelly/sensor.py#L85